app/uk/gov/hmrc/agentsubscription/connectors/AgentAssuranceConnector.scala

```diff
@@ -51,9 +49,7 @@ trait AgentAssuranceConnector {
 
 @Singleton
 class AgentAssuranceConnectorImpl @Inject() (val appConfig: AppConfig, http: HttpClient, metrics: Metrics)
-    extends AgentAssuranceConnector with HttpAPIMonitor with Logging {
-
-  override val kenshooRegistry: MetricRegistry = metrics.defaultRegistry
+    extends AgentAssuranceConnector with Logging {
```

Comment:
Could you pLease implement HttpAPIMonitor trait, so you do not have to implement timer with each connector.  Please have a look to other projects